ReleaseX.md

```diff
@@ -1,7 +1,7 @@
 # Release Notes for SCS Release X
 
 This document is work in progress for the upcoming Release X.
-Release 4 will be released in <Months/Year>. 
+Release 4 will be released in Months/Year.
```

Comment:
I know it has nothing to do with the docusaurus upgrade, but can we change this?
```suggestion
Release X will be released in Month/Year.
```